src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -155,6 +155,13 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
           WorkspaceService.props(workspaceServiceConstructor, userInfo),
           WorkspaceService.RevokeAllUserReadAccess(WorkspaceName(workspaceNamespace, workspaceName)))
       }
+    } ~
+    path("admin" / "validate" / Segment / Segment / Segment) { (workspaceNamespace, workspaceName, userSubjectId) =>
```

Comment:
this path for the endpoint feels kind of weird but I can't think of anything much better. suggestions?